app/controllers/explore_controller.rb

```diff
@@ -1,14 +1,15 @@
 # frozen_string_literal: true
-#= Controller for course functionality
+#= Controller for campaign/course functionality
 class ExploreController < ApplicationController
   respond_to :html
 
   def index
     # 'cohort' is the old name for campaign. We accept 'cohort' as an alternative
-    # parameter to keep old incoming links from breaking.
-    campaign = params[:campaign] || params[:cohort] || ENV['default_campaign']
-    @presenter = CoursesPresenter.new(current_user, campaign)
-    return unless @presenter.campaign.nil?
-    raise ActionController::RoutingError.new('Not Found'), 'Campaign does not exist'
+    # Redirect to new campaign overview page if a parameter is given, for backwards compatibility
+    if campaign = params[:campaign] || params[:cohort]
+      redirect_to campaign_path(campaign)
```

Comment:
Does this remove the ability to access the list of unsubmitted courses (the `NullCampaign` in `CoursesPresenter`)? We need some way to see that set of courses. It could even be a separate view and route that just renders the courses list with `campaign_param == 'none'`. Alternatively, we could continue the same approach and special-case the unsubmitted set as it's own fake campaign, and handle that in campaign controller to show it like any other campaign.